src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $calendar-gap;
+  padding: 2 * $calendar-padding $calendar-padding;
+  margin: auto;
+
+  &__day {
+    display: flex;
+    @include square ($day-size);
+    background-color: #eee;
+    border: 1px solid black;
+    align-items: center;
+    justify-content: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: background-color 0.5s linear, transform 0.5s linear;
+    cursor: pointer;
+
+    @for $index from 1 to 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+      }
+    }
```

Comment:
```suggestion
    @for $index from 1 through 31 {
      &:nth-child(#{$index})::before {
        content: "#{$index}";
      }
    }
```
this month looks like this

<img width="736" alt="image" src="https://user-images.githubusercontent.com/55802630/174733600-8e434228-cd25-4815-a667-f886b66400c5.png">